src/scripts.js

```diff
@@ -9,16 +9,16 @@ import { enableSkipLinks } from './js/utils';
 const headerDOM = document.querySelector('[data-template="global-header"]');
 const footerDOM = document.querySelector('[data-template="footer"]');
 const constructionModalDOM = document.querySelector('[data-template="construction-modal"]');
-const header = {
+const globalVariables = {
```

Comment:
@aevanson - is there a way we can truly globalize this variable so that we don't need to keep defining it with each component?

Reply:
Well, it's "only" the components that are used on the About Us page (that use icons) which are affected. This is because they could be on the root page or a sub-page and the relative path changes depending on which one it is. So the changes you've made for the gallery component or the modal-dynamic one (for example) aren't required. It should only be the header, mini-cart, and the construction modal components that would require this approach.

Regardless, the way the handlebars precompiled templates are set up, you have to pass in a data object if you want the loaded template to use dynamic values. Not sure if there's a better way than what you have here (in scripts.js)

Reply:
@aevanson so... approved?